core-web/apps/dotcms-ui/src/app/portlets/dot-templates/dot-template-create-edit/dot-template-create-edit.component.html

```diff
@@ -18,9 +18,8 @@
             <dot-template-builder
                 [didTemplateChanged]="didTemplateChanged.value"
                 [item]="vm.working"
-                (updateTemplate)="updateWorkingTemplate($event)"
+                (updateTemplate)="updateTemplate($event)"
```

Comment:
Yes, it was wrong. `updateTemplate` was changing the whole state instead of the working one. Which is the one we are suppose to update, not the original